gems/aws-sdk-cognitoidentity/lib/aws-sdk-cognitoidentity/customizations/cognito_identity_credentials.rb

```diff
@@ -0,0 +1,132 @@
+require 'set'
+require 'securerandom'
+require 'base64'
+
+module Aws
+  module CognitoIdentity
+    # An auto-refreshing credential provider that
+    # represents credentials retrieved from STS Web Identity Federation using
+    # the Amazon Cognito Identity service.
+    #
+    # This provider gets credentials using the
+    # {Client.get_credentials_for_identity} service operation, which
+    # requires either an `identity_id` or an `identity_pool_id` (Amazon Cognito
+    # Identity Pool ID), which is used to call {Client.get_id} to
+    # obtain an `identity_id` automatically.
+    #
+    # In addition, if this credential provider is used to provide authenticated
+    # login, the `logins` map may be set to the tokens provided by the
+    # respective identity providers. See {constructor} for an example on
```

Comment:
Is this `{constructor}` tag linking correctly? I think in our Yard code we expect Constructor but do double check.

Reply:
This should actually be {#initialize}.  Updated.